src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -533,6 +533,16 @@ COOP_PINVOKE_HELPER(void, RhSetThreadExitCallback, (void * pCallback))
     g_threadExitCallback = (ThreadExitCallback)pCallback;
 }
 
+#else
```

Comment:
You can just delete the `#ifdef TARGET_UNIX` ifdef instead of duplicating the code.